utils/build.ps1

```diff
@@ -552,6 +661,14 @@ function Fetch-Dependencies {
     Download-Python $BuildArchName
   }
 
+  if ($AndroidSDKArchs.count -gt 0) {
+    $NDKURL = "https://dl.google.com/android/repository/android-ndk-r26b-windows.zip"
```

Comment:
Would it make sense to check/assert that `$AndroidNDKVersion` == `r26b` as we are hardcoding to the r26b URL/hash here?

Reply:
Good catch - this should be substituted by the variable.